frontend/appflowy_flutter/lib/mobile/presentation/database/board/widgets/mobile_hidden_groups_column.dart

```diff
@@ -150,88 +160,97 @@ class MobileHiddenGroup extends StatelessWidget {
 
     return BlocBuilder<BoardBloc, BoardState>(
       builder: (context, state) {
```

Comment:
We have a few `BoardBloc` builders, should we add some logic to `buildWhen` to stop rebuilds to happen unnecessarily?

Reply:
Removed one